Docs 2979: Review payment method URLs (#2023)
MultiSafepay/docs: content/getting-started/glossary/multisafepay-glossary.md

A software developer a merchant employs to work on the technical side of their integration.

### ecommerce platform
An online [platform](/integrations/ready-made/) merchants use for accepting payments, e.g. website, mobile app, PWAs. 

### fraud
When a person tries to receive products without paying for them by tricking a customer or merchant.

### integration

> Software merchants use to sell products and services and accept payments, e.g website, mobile app, [PWA](/payments/integrations/pwa/). Merchants may use [ecommerce platforms](/integrations/ready-made/) or build a custom integration.

Reviewer: ```suggestion
Software merchants use to sell products and services and accept payments, e.g website, mobile app, [PWA](/payments/integrations/pwa/). Merchants may use [ecommerce platforms](/integrations/ready-made/) or [build their own integration](/integrations/self-made/).
```
